libraries/js/package.json

```diff
@@ -31,16 +31,17 @@
   },
   "homepage": "https://github.com/ProjectLibertyLabs/siwf#readme",
   "devDependencies": {
-    "@eslint/js": "^9.13.0",
+    "@eslint/js": "^9.14.0",
     "@types/eslint__js": "^8.42.3",
-    "@vitest/coverage-v8": "^2.1.3",
-    "@vitest/ui": "^2.1.3",
-    "eslint": "^9.13.0",
+    "@vitest/coverage-v8": "^2.1.4",
+    "@vitest/ui": "^2.1.4",
+    "eslint": "^9.14.0",
     "prettier": "3.3.3",
-    "tsup": "^8.3.0",
+    "tsup": "^8.3.5",
     "typescript": "^5.6.3",
-    "typescript-eslint": "^8.11.0",
-    "vitest": "^2.1.3"
+    "typescript-eslint": "^8.13.0",
+    "vite": "^5.4.10",
```

Comment:
Is it tied to some change ? Since this an addition

Reply:
Yup. It was an undeclared dependency before. It was used, but was being pulled in just as a sub-dependency